packages/mui-base/src/useButton/useButton.ts

```diff
@@ -20,7 +20,7 @@ import { EventHandlers } from '../utils/types';
  *
  * - [useButton API](https://mui.com/base/api/use-button/)
  */
-export default function useButton(parameters: UseButtonParameters): UseButtonReturnValue {
+export default function useButton(parameters?: UseButtonParameters): UseButtonReturnValue {
```

Comment:
IMO it would be safer to specify it as:
```suggestion
export default function useButton(parameters: UseButtonParameters = {}): UseButtonReturnValue {
```
This way, `parameters` will always be defined inside the body. Plus, it's shorter :)

Reply:
You are right ~ I totally overcomplicated this 😅